Documentation/building/freebsd-instructions.md

```diff
@@ -19,43 +19,36 @@ Install the following packages for the toolchain:
 
 - bash
 - cmake
-- llvm35 (should be installed automatically as dependency of clang35)
-- clang35
+- llvm37 (includes LLVM 3.7, Clang 3.7 and LLDB 3.7)
 - libunwind
 - gettext
 
-Note: LLDB is not yet available in the pkg repository on FreeBSD. The LLDB plugin is currently not being build on FreeBSD. Instructions for this will be added later.
-
 To install the packages you need:
 
-`janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake clang35 libunwind gettext`
+```sh
+janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake libunwind gettext llvm37
+```
 
-You now have all the required components.
+Want to user a lower version of Clang / LLVM? See [Note on Clang/LLVM versions](#note-on-clangllvm-versions).
```

Comment:
Can you rephrase that sentence? The style of speech does not fit the rest of the text and it is also not really what you expect in a technical documentation.